jujugui/static/gui/src/app/components/deployment-flow/deployment-flow.js

```diff
@@ -177,6 +178,16 @@ YUI.add('deployment-flow', function() {
     },
 
     /**
+      Store the provided SSH key in state.
+
+      @method _setSSHKey
+      @param {String} key The SSH key.
+    */
+    _setSSHKey: function(key) {
+      this.setState({sshKey: key});
```

Comment:
Just hit a thought on should we be looking to leverage ssh keys in LP or GH APIs for any of this?